dist/windows/strawberry.nsi.in

```diff
@@ -145,13 +145,30 @@ UninstPage custom un.LockedListPageShow
 !insertmacro MUI_UNPAGE_FINISH
 
 !insertmacro MUI_LANGUAGE "English" ;first language is the default language
+!insertmacro MUI_LANGUAGE "Italian"
 
 Name "${PRODUCT_NAME}"
 
+Caption "${PRODUCT_NAME} ${PRODUCT_DISPLAY_VERSION}"
+
 OutFile "${PRODUCT_NAME_SHORT}Setup-${PRODUCT_DISPLAY_VERSION}${build_type}-${compiler}-${arch}.exe"
 
 InstallDir "${PRODUCT_INSTALL_DIR}"
 
+LangString CHECKING_RUNNING_PROGRAMS  ${LANG_ENGLISH} "Checking for running programs:"
+LangString SEARCHING_RUNNING_PROGRAMS ${LANG_ENGLISH} "Searching for running programs..."
+LangString NO_PROGRAMS_TO_CLOSE       ${LANG_ENGLISH} "No programs need to close."
+LangString DOWNLOADING                ${LANG_ENGLISH} "Downloading..."
+LangString ALREADY_INSTALLED          ${LANG_ENGLISH} "${PRODUCT_NAME} is already installed. $\n$\nClick `OK` to remove the previous version or `Cancel` to cancel this upgrade."
+LangString UNINSTALL_LINK_DESCRIPTION ${LANG_ENGLISH} "Uninstall ${PRODUCT_NAME}"
+
+LangString CHECKING_RUNNING_PROGRAMS  ${LANG_ITALIAN} "Controllo programmi in esecuzione:"
+LangString SEARCHING_RUNNING_PROGRAMS ${LANG_ITALIAN} "Ricerca programmi in esecuzione.."
+LangString NO_PROGRAMS_TO_CLOSE       ${LANG_ITALIAN} "Nessun programma da chiudere."
+LangString DOWNLOADING                ${LANG_ITALIAN} "Download..."
+LangString PROGRAM_ALREADY_INSTALLED  ${LANG_ITALIAN} "${PRODUCT_NAME} è già installato.$\n$\nSeleziona `OK` per rimuovere la versione installata o `Annulla` per annullare questo aggiornamento."
```

Comment:
Same again: The variable for English is: `ALREADY_INSTALLED` while the variable for Italian is `PROGRAM_ALREADY_INSTALLED`